### PR TITLE
Show path to cache initialization

### PR DESCRIPTION
**Pull request by gizmomogwai**

### CODE REVIEW

File: projectile.el [hunk 1]
```diff
@@ -1786,7 +1786,7 @@ https://github.com/abo-abo/swiper")))
     ;; Calculate the list of files.
     (when (null files)
       (when projectile-enable-caching
-        (message "Projectile is initializing cache..."))
+        (message (format "Projectile is initializing cache in %s ..." (projectile-project-root))))
```
Comment (bbatsov) [inline projectile.el]: `format` is not needed here, as `message` also supports formatting options. `in` should probably be `for` and `(projectile-project-root)` can simply be replaced with a reference to the `project-root` param.

**Comment by gizmomogwai** [comment]
thanks for the improvements.

**Comment by gizmomogwai** [comment]
could you merge this, or should i rebase or abandon the commit?

**Comment by bbatsov** [comment]
Merge it is. Thanks!

**Pull request merged**